common/mariadb/values.yaml

```diff
@@ -20,6 +19,17 @@ expire_logs_days: 10
 character_set_server: 'utf8'  # Should be utf8mb4, but we started with this
 collation_server: 'utf8_general_ci'
 
+db_performance_schema: # Enabling performance schema only (without enabling instruments for data collections).
+                       # Instrument can be enabled separately. Only Performance Schema Enablement does not cause any
+                       # overhead.
+  enabled: true
+  
+db_performance_schema_instrument: # Enabling performance schema instruments. 
+                                  # Please enable the db_performance_schema in order to enable the db_performance_schema_instrument.
+  enabled: false
+  # default : false
```

Comment:
I don't think we need to write the default here, as this `values.yaml` is the default - a parent chart would have to change the value in their values file to move away from the default.

Reply:
I don't get that. Who is the intended audience?

As said, the default is, what the setting right above defines. Everything non-default would be handled in parent charts and their values files.

Reply:
It means this value has to be false for production

Reply:
I am reading that as there is a key `default`, which is assumed to be `false`, but since it is the same as not setting it, there is a comment to indicate the existence (and type).

I would rather go with adding a comment to `enabled: false` explaining why it is on false by default

Reply:
Thanks for the feedback! I will do the update with the next PR.

But that was the explanation: # Instrument can be enabled separately. Only Performance Schema Enablement does not cause any overhead.